File: src/main/java/load/objects/AWSCredentials.java
package load.objects;

/**
 * Created by user on 17.03.2017.
 */
public class AWSCredentials {
    String accessKeyId;
    String secretAccessKey;
    String sessionToken;

    @Override
    public String toString() {
        return "AWSCredentials{" +
                "accessKeyId='" + accessKeyId + '\'' +
                ", secretAccessKey='" + secretAccessKey + '\'' +
                ", sessionToken='" + sessionToken + '\'' +
                '}';
    }

    public String getAccessKeyId() {
        return accessKeyId;
    }

    public void setAccessKeyId(String accessKeyId) {
        this.accessKeyId = accessKeyId;
    }

    public String getSecretAccessKey() {
        return secretAccessKey;
    }

    public void setSecretAccessKey(String secretAccessKey) {
        this.secretAccessKey = secretAccessKey;
    }

    public String getSessionToken() {
        return sessionToken;
    }

    public void setSessionToken(String sessionToken) {
        this.sessionToken = sessionToken;
    }
}

File: src/main/java/load/gatling/scala/AWSCredentials.scala
class AWSCredentials {
  var accessKeyId: String = null //required for request and for sign
  var secretAccessKey: String = null //required for sign
  var sessionToken: String = null //required for request
  override def toString: String = {
    return "AWSCredentials{" + "accessKeyId='" + accessKeyId + '\'' + ", secretAccessKey='" + secretAccessKey + '\'' + ", sessionToken='" + sessionToken + '\'' + '}'
  }

  def getAccessKeyId: String = {
    return accessKeyId
  }

  def setAccessKeyId(accessKeyId: String) {
    this.accessKeyId = accessKeyId
  }

  def getSecretAccessKey: String = {
    return secretAccessKey
  }

  def setSecretAccessKey(secretAccessKey: String) {
    this.secretAccessKey = secretAccessKey
  }

  def getSessionToken: String = {
    return sessionToken
  }

  def setSessionToken(sessionToken: String) {
    this.sessionToken = sessionToken
  }
}